app/cdap/text/text-en.yaml

```diff
@@ -3439,3 +3439,21 @@ features:
   ConnectionsList:
     labels:
       loadToGrid: 'Wrangle'
```

Comment:
add all items in alphabetical order and cross-check for recently added items.

Reply:
everything label in menu should be in Titlecase